src/stream-tcp.c

```diff
@@ -418,7 +418,9 @@ void StreamTcpInitConfig(char quiet)
         SCLogConfig("stream \"memcap\": %"PRIu64, SC_ATOMIC_GET(stream_config.memcap));
     }
 
-    ConfGetBool("stream.midstream", &stream_config.midstream);
+    int imidstream;
+    ConfGetBool("stream.midstream", &imidstream);
```

Comment:
Thanks for this...
Should `ConfGetBool` use a bool instead ?
Maybe not for this PR, but we can file a redmine ticket about this

Reply:
Well, it could be good to modify ConfGetBool so.
However, it will result in massive changes in corresponding variables/structure members types. Also, it would be reasonable to get rid of conditions (something ==/!= TRUE) and change them to (something) or (!something)
So the proposed modification is good but it should be made in the scope of a special ticket and very carefully.

Reply:
Thanks for your comment